omod/src/main/web/dashboardwidgets/dashboardwidgetscommons.service.js

```diff
@@ -127,4 +127,32 @@ export default class WidgetsCommons {
         return Boolean(obj.concept);
     }
 
+    /**
+     * Returns date translated according to preferred locale when Date Format displays the month as an abbreviation (MMM).
+     * Ex: Date: 02-Jan-2021 Format: DD-MMM-YYYY Locale: fr_FR --> Return: 02-janv.-2021
+     *
+     * @param {object} date Input Date
+     * @param {string} format Date Format
+     * @param {string} locale The preferred locale
+     */
+    dateFormatLocale(date, format, locale) {
```

Comment:
@icrc-ifernandes, coud you rename this as `formatDate(date, format, locale)`?

Reply:
@mks-d method renamed as requested.